kyuubi-hive-jdbc-shaded/pom.xml

```diff
@@ -118,6 +131,10 @@
                             <pattern>org.apache.zookeeper</pattern>
                             <shadedPattern>${kyuubi.shade.packageName}.org.apache.zookeeper</shadedPattern>
                         </relocation>
+                        <relocation>
+                            <pattern>org.slf4j</pattern>
```

Comment:
I'm afraid we can not relocate classes under `org.slf4j`, because the relocation break the log framework binding.

Reply:
good catch, reverted.